osgp/platform/osgp-domain-core/src/main/java/org/opensmartgridplatform/domain/core/entities/Device.java

```diff
@@ -193,6 +193,12 @@
     @Enumerated(EnumType.STRING)
     private IntegrationType integrationType = IntegrationType.WEB_SERVICE;
 
+    @Column
+    private Integer cellId;
+
+    @Column
+    private Integer btsId;
```

Comment:
These are updated together with the `networkAddress`. Would it make sense to keep them a bit closer together in the source file?

Reply:
agree.